packages/discord.js/src/managers/GuildChannelManager.js

```diff
@@ -96,6 +96,24 @@ class GuildChannelManager extends CachedManager {
     return super.resolveId(channel);
   }
 
+  /**
+   * Adds the target channel to channel's followers.
+   * @param {TextChannelResolvable} channel The channel the follower should be added to
```

Comment:
This isn't a `TextChannelResolvable` - this accepts a `TextChannel` or a `Snowflake`. We need a `NewsChannel` or a `Snowflake`.